### PR TITLE
[Android] Add android aar package

### PR DESCRIPTION
**Pull request by DefTruth**
### PR types(PR类型)

 Android

### Describe

- add fastdeploy android aar package
  - [x] init fastdeploy android jar package  
  - [x] support INT8 option for lite backend  
  - [x] use CopyFromCpu for lite backend
  - [x] add lite backend to `valid_cpu_backends` for all paddle models
       - [x] add `Backend::Lite` to paddle models
       - [x] test  `Backend::Lite` for  paddle models
  - [x] bind ppclas series models with JNI and Java
       - [x] 10+ PaddleClas Model 
  - [ ] bind PP-OCRv1, PP-OCRv2 and PP-OCRv3 with JNI and Java  
  - [ ] bind ppdet series models with JNI and Java
       - [x] PicoDet s/xs/m/l  
       - [ ] PPYOLOE/PPYOLOE+  
       - [ ] ... 
  - [ ] bind all external yolo series models(v5,v6,v7,x,v5lite,...) with JNI and Java  
  - [ ] bind pptinypose series models with JNI and Java
  - [ ] bind text/uie models with JNI and Java
  - [ ] bind ppseg series models with JNI and Java
  - [ ] bind external facedet models (yolov5face,retinaface,scrfd,ultraface,...) with JNI and Java
  - [ ] ......

### CODE REVIEW

File: examples/vision/ocr/PP-OCRv2/cpp/infer.cc [hunk 1]
```diff
@@ -45,6 +45,7 @@ void InitAndInfer(const std::string& det_model_dir, const std::string& cls_model
     std::cerr << "Failed to initialize PP-OCR." << std::endl;
     return;
   }
+  std::cout << "Backend::Lite initialized done!\n";
```
Comment (jiangjiajun) [inline examples/vision/ocr/PP-OCRv2/cpp/infer.cc]: 删除

File: examples/vision/ocr/PP-OCRv3/cpp/infer.cc [hunk 2]
```diff
@@ -45,6 +45,7 @@ void InitAndInfer(const std::string& det_model_dir, const std::string& cls_model
     std::cerr << "Failed to initialize PP-OCR." << std::endl;
     return;
   }
+  std::cout << "Backend::Lite initialized done!\n";
```
Comment (jiangjiajun) [inline examples/vision/ocr/PP-OCRv3/cpp/infer.cc]: 删除

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: examples/vision/ocr/PP-OCRv2/cpp/infer.cc (hunk 1 above)
Comment (DefTruth) [reply examples/vision/ocr/PP-OCRv2/cpp/infer.cc]: fixed

File: examples/vision/ocr/PP-OCRv3/cpp/infer.cc (hunk 2 above)
Comment (DefTruth) [reply examples/vision/ocr/PP-OCRv3/cpp/infer.cc]: fixed

**Review by jiangjiajun — APPROVED** [review]

**Pull request merged**